src/main/java/org/opentripplanner/model/plan/Leg.java

```diff
@@ -9,6 +9,7 @@
 import java.util.stream.Stream;
 import javax.annotation.Nullable;
 import org.locationtech.jts.geom.LineString;
+import org.opentripplanner.ext.fares.model.FareProductUse;
```

Comment:
The API classes should be moved into `org.opentripplanner.model.plan.fare`, the api classes should not have dependencies to the sandbox.

Reply:
Just this one class or all of them? I was planning to do a follow up PR with lots of clean ups where I move stuff into the right place.

Reply:
I've moved all fare model classes in 204751f9432ae4568abaec92dff8d6442ebb9283

Reply:
We had a quick chat 1-on-1 and I've implemented the result of this in https://github.com/opentripplanner/OpenTripPlanner/pull/4917/commits/204751f9432ae4568abaec92dff8d6442ebb9283